app/tests/test_accounts.yaml

```diff
@@ -0,0 +1,7 @@
+accounts:
```

Comment:
create a manifests dir under tests and place there

Reply:
done